amr-wind/equation_systems/vof/SplitAdvection.H

```diff
@@ -0,0 +1,24 @@
+#ifndef SplitAdvection_H
```

Comment:
Seems like we can combine `lagrangian_advection.H` and `SplitAdvection.H` into a single header.

Reply:
Sure, but I will also add a eulerian_advection file so I prefer to keep them separate